fe1-web/src/features/rollCall/components/EventRollCall.tsx

```diff
@@ -57,20 +56,25 @@ const EventRollCall = (props: IPropTypes) => {
         );
         return;
       }
-      requestOpenRollCall(event.idAlias).catch((e) =>
+      requestReopenRollCall(event.idAlias).catch((e) =>
```

Comment:
Wouldn't it be clearer to split this function into 2 ? Having a function called `onOpenRollCall` that opens and reopens roll call based on a boolean value seems quite odd to me.

Reply:
Yep, seems odd to me too !